.github/workflows/build_conda_windows.yml

```diff
@@ -80,11 +80,20 @@ jobs:
           ref: ${{ inputs.test-infra-ref }}
           path: test-infra
       - uses: ./test-infra/.github/actions/set-channel
+      - uses: ./test-infra/.github/actions/setup-ssh
```

Comment:
nit: But if we do this enough for windows perhaps it'd be good to just split this out into it's own action so that we don't have to copy / paste this everywhere we use this